nasa/cFE
# Fix #540, User's guide scrub

**Describe the contribution**
Documentation/comments only included in this change.

Commit https://github.com/nasa/cFE/pull/555/commits/784fe0965fa7b042c6334b8e090bc636e5a254e4
 - Fixes #540 users guide scrub (see issue for individual changes)
 - Fixes #403 all warnings
 - Fixes #536 by referencing header documentation (removes typo)

Commit https://github.com/nasa/cFE/pull/555/commits/0f606755d5464c7d87dd34746c8c78b9cc1bf874
 - Fixes #261

Commit https://github.com/nasa/cFE/pull/555/commits/7e248baf4192a452ead52f6a4e17af6e191d0970
- Fixes #267

Commit https://github.com/nasa/cFE/pull/555/commits/71df47f05543079eb219013851f91f394f77511c
- Fixes #499 

Commit https://github.com/nasa/cFE/pull/555/commits/0c00f71651237d1f0ab6ae7f750c0006008d9512
- Fixes #531

**Testing performed**
Steps taken to test the contribution:
1. Standard make/build steps to confirm code wasn't broken
1. make usersguide
1. confirmed build/doc/warnings.log (doxygen warnings) empty
1. cd build/doc/usersguide/latex
1. make  (to build the pdf)

**Expected behavior changes**
User's guide generates and and creates pdf without errors

**System(s) tested on**
 - Hardware: cFS Dev Server
 - OS: Ubuntu 18.04
 - Versions: master bundle w/ this commit

**Additional context**
Does not attempt to address #508 and #389 

**Third party code**
None

**Contributor Info - All information REQUIRED for consideration of pull request**
Jacob Hageman - NASA/GSFC
